yarn-project/end-to-end/src/e2e_p2p/gerousia.test.ts

```diff
@@ -17,7 +17,9 @@ import { P2PNetworkTest } from './p2p_network.js';
 
 // Don't set this to a higher value than 9 because each node will use a different L1 publisher account and anvil seeds
 const NUM_NODES = 4;
-const BOOT_NODE_UDP_PORT = 40600;
```

Comment:
This file don't exists on master anymore 👀

Reply:
oh dear